services/filemanager/filemanager.go

```diff
@@ -46,6 +46,10 @@ func New(settings *SettingsT) (FileManager, error) {
 		return &MinioManager{
 			Config: GetMinioConfig(settings.Config),
 		}, nil
+	case "DIGITAL_OCEAN_SPACES":
+		return &DOspacesManager{
```

Comment:
Are there any difference between minioManager.go and digitalOceanSpacesManager.go? If not can we remove it and just return and use minioManager itself for spaces here

Reply:
no, there are no differences .. i just changed the variable names for spaces. Yes, ill make those changes and return minioManager for spaces too